test/installPSRP.sh

```diff
@@ -17,10 +17,10 @@ chmod +x ./installpowershell.sh
 
 redmondpassword=$1
 isMacOS=false
-omiversion="1.1.0-28"
+omiversion="1.1.0-52"
 psrpversion="3"
-powershellDir="/opt/microsoft/powershell/6.0.0-alpha.13"
-powershellDirForMac="/usr/local/microsoft/powershell/6.0.0-alpha.13"
+powershellDir="/opt/microsoft/powershell/6.0.0-alpha.14"
+powershellDirForMac="/usr/local/microsoft/powershell/6.0.0-alpha.14"
 realdataDir="//wsscnafiler43/ostcdata$"
```

Comment:
The //wsscnafiler43/ostcdata$ link is going to change next week. You should always use the DFS name as that will not change.

DFS: \\redmond\wsscfs\OSTCData
OLD: \\wsscnafiler43.redmond.corp.microsoft.com\ostcdata$\OSTCData
NEW:  \\cdmbuildsna01.redmond.corp.microsoft.com\ostcdata$\OSTCData